packages/storybook/stories/link.mdx

```diff
@@ -0,0 +1,44 @@
+import { RouteLink } from '@equisoft/design-elements-react';
+import { ArgTypes, Canvas, Meta } from '@storybook/blocks';
+import * as RouteLinkStories from './link.stories';
+
+<Meta of={RouteLinkStories} />
+
+# Link
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+Links are a combination of a web address (URL) and a clickable element (often a word or phrase, sometimes an image), that brings the user somewhere and that doesn’t affect the back or front-end [1][2]. 
```

Comment:
C'est vrai mais pas au niveau de notre composant. Le composant présentement autorise seulement une icône et un texte. L'image n'est pas une possibilité avec ce composant. Pas nécessaire de faire mention du front-end et du back end.

"_The Link component enables users to seamlessly navigate between pages, both within and outside the application._"